tests/func/test_pull.py

```diff
@@ -78,7 +78,7 @@ def _adapt_row(row):
 def schema():
     return {
         "id": {"type": "UInt64"},
-        "sys__rand": {"type": "Int64"},
+        "sys__rand": {"type": "UInt64"},
```

Comment:
For consistency with `DataTable.sys_columns` here? 🤔 

https://github.com/iterative/datachain/blob/976b5fd1b27d33f40fc25d88a96cc5ddeda378cd/src/datachain/data_storage/schema.py#L279

Reply:
Oh, by the way: on SQLite it looks like the maximum `INTEGER` is a **signed** 64-bit number. 🙃